### ISSUE TITLE
Plans for 1.21?

### ISSUE THREAD

**Issue by Lou9896** [mactso/Regrowth#52]
Are there any plans to update the mod for 1.21?

**Comment by mactso** [comment]
Yes.   just had a *hell* of a last 12 months.  Multiple stays in the hospital and two natural disasters without electricity for a week each time.    I hope to do the 1.21 push for all my mods starting very soon.  I'm catching up on easy to fix bugs right now then it's go time.

**Comment by mactso** [comment]
Okay, Good News, Regrowth for 1.21.1 for forge is uploaded to curseforge and modrinth.
1.21.1 for Fabric is probably a few weeks away.

I'm going to leave this open until I get the Fabric version done too.
Cheers,
Two Guys,
MacTso
